app/src/ui/lib/install-cli.ts

```diff
@@ -0,0 +1,64 @@
+import * as Fs from 'fs'
+import * as Path from 'path'
+
+const runas: (
+  command: string,
+  args: ReadonlyArray<string>,
+  options?: { admin: boolean }
+) => number = require('runas')
+
+/** The path for the installed command line tool. */
+export const InstalledCLIPath = '/usr/local/bin/github'
+
+/** The path to the packaged CLI. */
+const PackagedPath = Path.resolve(__dirname, 'static', 'github.sh')
```

Comment:
@joshaber Is this gonna be static across versions on macOS? I don't know how that works at all